deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBDeployDAOImpl.java

```diff
@@ -201,23 +203,28 @@ public UpdateStatement genInsertStatement(DeployBean deployBean) {
     public List<DeployBean> getAcceptedDeploys(String envId, Interval interval, int size)
         throws Exception {
         ResultSetHandler<List<DeployBean>> h = new BeanListHandler<>(DeployBean.class);
-        String
-            typesClause =
-            QueryUtils.genEnumGroupClause(StateMachines.AUTO_PROMOTABLE_DEPLOY_TYPE);
+        List<Object> params = new ArrayList<>();
+        params.add(envId);
+        params.addAll(StateMachines.AUTO_PROMOTABLE_DEPLOY_TYPE.stream().map(Enum::name).collect(Collectors.toList()));
```

Comment:
Optional: `StateMachines.AUTO_PROMOTABLE_DEPLOY_TYP` is not user provided, so it doesn't have to be parameterized.